src/ResourceManagement/ResourceManager/Generated/SubscriptionClient.cs

```diff
@@ -93,7 +93,7 @@ public SubscriptionClient(RestClient restClient) : base(restClient)
         /// </summary>
         protected override void Initialize()
         {
-            Operations = new Operations(this);
+            //Operations = new Operations(this);
```

Comment:
After commented out, where does this Operations get init?

Reply:
It's like autorest issue here. Operations is used by ResourceManagementClient. So I just comment out here.

Reply:
If so, check whether you need to remove the Operations property as well, like the file before your upgrade.